default/python/universe_generation/starnames.py

```diff
@@ -177,7 +177,7 @@ def name_star_systems(system_list):
     indiv_systems = []
 
     # remove groups with only one non-deep-system
-    for groupindex, group_list in star_groups.items():
+    for groupindex, group_list in list(star_groups.items()):
```

Comment:
Why do we need `list` wrapper for iteration?

Reply:
Because `star_groups` is mutates in the loop which result in the error `RuntimeError: dictionary changed size during iteration`